crtool/js/analytics.js

```diff
@@ -0,0 +1,88 @@
+const jQuery = window.jQuery;
```

Comment:
It'd be really great, if you have time, to remove jQuery from this repo. It looks like the ways its used would be relatively easy to reproduce in vanilla JS.

If not, the next best thing would be to pull it in as an npm dependency (upgrading to the latest version, since we don't need to support IE<11), and importing it as a module.

Reply:
Switched to npm dependency .